vuepress/docs/next/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebarDepth: 2
+redirectFrom: /next/tutorials/ecr/tutorials/create-ecr-bundle-from-git.html
+---
+# Build and Publish a Simple Bundle
+
+## Overview
+This tutorial shows you how to create a simple Entando bundle and deploy it into the Entando Component Repository. This involves manually defining a bundle with a single widget, checking your bundle artifacts into git, apply the Entando bundle custom resource to Kubernetes, and then installing the bundle into your application. 
+
+## Prerequisites
+* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) tool to verify you have the necessary prerequisites in place (e.g. Java, npm, git, etc.). You will also need your git credentials, an available git repository, and an Entando instance. Some of the following steps can be simplified by using the `ent prj` command and its publication system (pbs) convenience methods. The manual steps are also provided in those cases.
```

Comment:
Might be worth reproducing the CLI command here too but maybe not. Unsure about my thinking on that one.